doc/news/_preparation_next_release.md

```diff
@@ -105,7 +105,18 @@ The text below summarizes updates to the [C (and C++)-based libraries](https://w
 Bindings allow you to utilize Elektra using [various programming languages](https://www.libelektra.org/bindings/readme). This section keeps
 you up to date with the multi-language support provided by Elektra.
 
-### <<Binding1>>
+### Java binding
+
+- Renamed zero argument static factory method `Key::createNameless` to `Key::create`. To migrate to this change, just update calling code to use the new method name.
+- Removed signatures publishing the error key based error handling approach to the Java binding consumer. Such arguments are now created implicitly by the binding. In case of an exceptional state, appropriate exceptions are thrown. Such exceptions provide access to the underlying key containing warning and error information as meta data. Affected signatures:
+  - Removed `KDB::open(Key)`, please update usages by calling `KDB::open()` instead.
+  - Removed `KDB::open(KeySet, Key)`, please update usages by calling `KDB::open(KeySet)` instead.
+  - Removed `KDB::close(Key)`, please update usages by calling `KDB::close()` instead.
```

Comment:
How can I get warnings then?

Reply:
A very interesting question, i have not considered ;) With these changes, you only get warnings, if there is also an error. (via the KDBException)
To support returning warnings i will introduce a key parameter like `warningsKey`.

Reply:
> I think we can merge and adjust the small things in a soon follow-up PR. @tucek is this okay for you?

@markus2330 i think all things, even the small ones, should be taken care of now. (see https://github.com/ElektraInitiative/libelektra/pull/3790)
If not, i think we should do it within this PR. (I can always rebase my next PR without any effort)